fe/lib/components/JobCategorySelect/JobCategorySelectInput.tsx

```diff
@@ -15,20 +15,23 @@ interface Props {
   onSelect: (jobCategory: AnyJobCategory, type: 'parent' | 'child') => void;
   tone: ComponentProps<typeof Dropdown>['tone'];
   hideLabel?: boolean;
+  initialJobCategoryId?: string;
 }
 
 const JobCategorySelectInput = ({
   jobCategories,
   onSelect,
   hideLabel,
+  initialJobCategoryId,
   ...restProps
 }: Props) => {
   const [selectedParentCategoryId, setSelectedParentCategoryId] = useState('');
   const [selectedChildCategoryId, setSelectedChildCategoryId] = useState('');
   const [childCategories, setChildCategories] = useState<AnyJobCategory[]>();
+  const [isInitialJobCategory, setIsInitialJobCategory] = useState<boolean>();
 
   useEffect(() => {
-    if (selectedParentCategoryId !== '') {
+    if (selectedParentCategoryId !== '' && !isInitialJobCategory) {
```

Comment:
Do we know why these are `useEffect`s instead of just being in the `onChange` handlers? That would avoid the weird `initialJobCategory` state

Reply:
It is on initial load. onChange would get triggered when we select a different option.

Reply:
Right, but do we *want* this to run on initial load? It seems like the `isInitialJobCategory` is just there to suppress it

Reply:
Hmm, good point. Let me try that

Reply:
Good pick up. There was no need of that useEffect.